payloads/ragdoll.py

```diff
@@ -27,16 +27,16 @@ def start(self):
         """
         self.profile['results'] = []
         print('[*] Sending beacon for %s' % self.profile.get('paw', 'unknown'))
-        try:
-            while True:
+        while True:
```

Comment:
what is the exit condition for this loop?

Reply:
As in does it ever exit? From my knowledge, and checking git history, this agent does not exit from an error once in operational loop.

My change here is to revert it how it was originally, as I didn't intend to change that particular policy/behavior in the last commit I did.